src/parameterized/LKJL.jl

```diff
@@ -38,17 +50,24 @@ function logdensity(d::LKJL{k}, L::Union{LinearAlgebra.AbstractTriangular, Diago
     η = d.η
     # z = diag(L)
     # sum(log.(z) .* ((k:-1:1) .+ 2*(η-1)))
-    c = k + 1 + 2(η - 1)
+
+    # Note: https://github.com/cscherrer/MeasureTheory.jl/issues/100#issuecomment-852428192
+    c = k + 2(η - 1)
     @tullio s = (c - i) * log(L[i,i])
     return s
 end
 
 using TransformVariables
 
-TransformVariables.as(::LKJL{k}) where {k} = TransformVariables.CorrCholeskyFactor(k)
+TransformVariables.as(::LKJL{k}) where {k} = CorrCholeskyLower(k)
 
+# TODO: I think this is wrong
```

Comment:
Not certain. Why do you think this is wrong?

Reply:
This starts with Lebesgue measure in ℝⁿ, then stretches the space using the transform. I think what we end up with will be an artifact of our preference to use ℝⁿ, instead of whatever is natural for the space. There must be an easy way to check this sort of thing, but I haven't thought of it yet.

Reply:
Oh hm, then yes, I think this is not quite correct. Because would this not also imply that one could evaluate `logdensity(Lebesgue(ℝ)^d, L)` for lower triangular `L`, i.e. that our samples are elements of `ℝ^d`?

Otherwise, wouldn't the base measure be `Lebesgue(ℝ)^(k,k)`, or however that is written, with the constant normalization factor?

Reply:
I think it's more complicated than that, since we need to match the dimension. For example,
```julia
julia> d = LKJL(3, 2.0)
LKJL{3}((η = 2.0,))


julia> rand(d)
3×3 LowerTriangular{Float64, Matrix{Float64}}:
  1.0         ⋅         ⋅ 
  0.232191   0.97267    ⋅ 
 -0.489373  -0.595999  0.636631
```
This looks like it has 5 degrees of freedom, but in reality,

```julia
julia> dimension(as(d))
3
```

Reply:
`Pushforward` looks like this:
```julia
struct Pushforward{F,M} <: AbstractMeasure
    f::F
    μ::M
    logjac::Bool
end
```

It might work to use this transform with `logjac=false`. I haven't tried that before, but... maybe?

Reply:
Oh, that's what I have :laughing: 

Ok maybe this is right. I guess the check is that the logdensity of the base measure should be constant on tranformed values.